splinepy/io/ioutils.py

```diff
@@ -97,3 +97,50 @@ def abs_fname(fname):
         fname = os.path.abspath(fname)
 
     return fname
+
+
+def strip_tabs(fname, overwrite=True, tab_expand=2):
+    """Replaces tabs in a text file with spaces
+
+    Parameters
+    ----------
+    fname : string
+      Filename
+    overwrite : bool
+      Inplace modification, otherwise new file with prefix copy is created
+    tab_expand : int
+      Determines how many spaces to be set for each tab
+
+    Returns
+    -------
+    None
+    """
+    if overwrite:
+        out_name = fname
+    else:
+        out_name = "copy." + fname
+    with open(fname) as inputFile:
+        file_contents = inputFile.read()
```

Comment:
`read()` may take too big memory if it tries to read a giant file. loop instead?

Reply:
Looping is difficult if the file is written back to the same file.

The only thing one could do is to do the loop and write it to the copy_fname file and depending on whether or not the file is to be overwritten move the copy_file to the original file location. Is there a possibility that these irit files get so big that this might be a problem?

Reply:
After looking at the code in `SplineLib/Sources/InputOutput/irit.cpp`, they also read in the whole file at once. So even if we do some workaround for not reading the entire file completely, the problem persists in the `irit` importer.

Reply:
sounds like a `won't fix`